app/queries/vacancy_filter_query.rb

```diff
@@ -31,6 +31,25 @@ def call(filters) # rubocop:disable Metrics/AbcSize
     built_scope
   end
 
+  def add_organisation_type_filters(filters, built_scope)
+    return built_scope unless filters[:organisation_types].present?
+
+    establishment_code_filter = []
+    establishment_name_filter = []
+
+    if filters[:organisation_types].include?("Academy")
+      %w[10 11].each { |code| establishment_code_filter << code }
+      ["Academies", "Free Schools"].each { |name| establishment_name_filter << name }
```

Comment:
You can avoid the looping using`concat`.
```suggestion
     establishment_code_filter.concat  %w[10 11]
     establishment_name_filter.concat ["Academies", "Free Schools"]
```

Reply:
Ah yes, perfect. Rubocop made me use push instead of concat but either way is a big improvement, ty!